de_DE/webapp/src/main/webapp/i18n/vivo_all_de_DE.properties

```diff
@@ -906,3 +906,5 @@ create_and_link_type_webpage=Webpage
 claim_publications_by=Claim publications by
 claim_publications_by_doi=DOI
 claim_publications_by_pmid=PubMed ID
+
+term_capitalized = Begriff
```

Comment:
Can you please add this new property under the "miscellaneous visualization templates ( /templates/freemarker/visualization)" section:
https://github.com/vivo-project/VIVO-languages/blob/master/webapp/src/main/webapp/i18n/vivo_all_de_DE.properties#L573-L582

Reply:
i did it